### ISSUE TITLE
Kentico unit tests throw null reference exception when ASP.NET Core 6 project has this nuget package installed

### ISSUE THREAD

**Issue by ttlaare** [heywills/StagingConfigurationModule#10]
A null reference exception is thrown when running unit tests with the test class inheriting from the Kentico libraries test base class 'UnitTests' and referencing an ASP.NET CORE 6 project with this NuGet package installed.

```csharp
    [TestFixture]
    public class Tests : UnitTests
    { 
        [Test]
        public void Test1()
        {
            Assert.Pass();
        }
    }
```

This exception will not get thrown when removing this Nuget package from the ASP.NET CORE 6 project.
Hotfix version: 13.0.68

**Comment by heywills** [comment]
Thank you @ttlaare . I'll take a look at this.

**Comment by heywills** [comment]
Hiya @ttlaare , this issue is fixed in v13.0.7. It was caused by the module trying to initialize with a null configuration parameter. 

https://www.nuget.org/packages/KenticoCommunity.StagingConfigurationModule/13.0.7

Here's the PR.
https://github.com/heywills/StagingConfigurationModule/pull/11

Will you let me know if it works for you?

**Comment by ttlaare** [comment]
Hi @heywills. Unit tests are running successfully now, so this is issue is fixed. Thanks!
